libs/designsystem/src/lib/components/form-field/form-field.component.spec.ts

```diff
@@ -48,6 +46,10 @@ describe('FormFieldComponent', () => {
     it('should have correct margin', () => {
       expect(spectator.element).toHaveComputedStyle({ 'margin-bottom': size('s') });
     });
+    it('should have elevation', () => {
```

Comment:
Consider moving to describe block `with slotted input` and adding similar test to `with slotted textarea` block...

Reply:
Makes sense 👍  
Done